### PR TITLE
Fixed wrong display of "limited support" tag on new results of

### PR DESCRIPTION
**Pull request by bereket-WMDE**
the lookup menu upon scroll

Bug: T270549

### CODE REVIEW

File: src/components/PropertyLookup.vue [hunk 1]
```diff
@@ -53,7 +53,14 @@ export default Vue.extend( {
 			}
 		},
 		async searchPropertiesOnScroll( options: SearchOptions ): Promise<void> {
-			const searchResults = await this.$store.dispatch( 'searchProperties', options );
+			let searchResults = await this.$store.dispatch( 'searchProperties', options );
+			searchResults = searchResults.map(
```
Comment (guergana) [inline src/components/PropertyLookup.vue]: this has the exact same content as [the watch](https://github.com/wmde/query-builder/blob/51673294a6fa2c0893c73a3752748ddf0de11692/src/components/PropertyLookup.vue#L71). It is better if it's extracted into a function.

**Review by guergana — CHANGES_REQUESTED** [review]

File: src/components/PropertyLookup.vue (hunk 1 above)
Comment (bereket-WMDE) [reply src/components/PropertyLookup.vue]: updated.

File: src/components/PropertyLookup.vue [hunk 2]
```diff
@@ -34,7 +34,7 @@ export default Vue.extend( {
 	data() {
 		return {
 			search: '',
-			searchResults: [],
+			searchResults: [] as object[],
```
Comment (guergana) [inline src/components/PropertyLookup.vue]: why cast type it to `object[]` instead of `SearchResult[]` ?

File: src/components/PropertyLookup.vue [hunk 3]
```diff
@@ -53,9 +53,19 @@ export default Vue.extend( {
 			}
 		},
 		async searchPropertiesOnScroll( options: SearchOptions ): Promise<void> {
-			const searchResults = await this.$store.dispatch( 'searchProperties', options );
+			let searchResults = await this.$store.dispatch( 'searchProperties', options );
+			searchResults = this.setTagForSearchResults( searchResults );
+
 			this.searchResults = this.searchResults.concat( searchResults );
 		},
+		setTagForSearchResults( searchResults: SearchResult[] ): object[] {
```
Comment (guergana) [inline src/components/PropertyLookup.vue]: same observation as before: `SearchResult[]` instead of object[] is a better type in this case.

**Review by guergana — CHANGES_REQUESTED** [review]

**Comment by bereket-WMDE** [comment]
@guergana good catch on the type casting comments!

**Review by guergana — APPROVED** [review]

**Pull request merged**